packages/contracts/docs/osx/01-how-it-works/01-core/index.md

```diff
@@ -26,8 +26,6 @@ In a nutshell, your Aragon OSx DAO consists of three pieces:
 
 The underlying smart contracts constitute **the core contracts** of the Aragon OSx DAO framework.
 
-The following graphic shows how an exemplary DAO setup, where the
```

Comment:
@Michael-A-Heuer shouldn't we rather complete the sentence?

Reply:
The image already has a caption below, I found this paragraph redundant:
<img width="1020" alt="Screenshot 2023-07-20 at 08 44 44" src="https://github.com/aragon/osx/assets/19923822/03c34ccd-dfe9-4bf3-b047-905c02cc2cd2">